components/server/src/auth/host-context-provider-impl.ts

```diff
@@ -58,9 +53,17 @@ export class HostContextProviderImpl implements HostContextProvider {
                 span.finish();
             }
         }, 1999);
+        this.disposables.push(updateDynamicHostsJob);
+
         this.initialized = true;
```

Comment:
let's moved this to first line of `init`. I think it's enough to signal a call to `init` was made. we don't need/want to wait for a first update.

Reply:
I thought this should signal a defined state? If we do this we don't need `assertInitialized` at all, no? :thinking:

Reply:
No. 

`assertInitialized` will throw if one would remove the `init` call by accident. Thus it needs to be checked in the public methods.